File: Python Exams/Basic Exam Collection/Basics Online Exam -  28 and 29 March 2020/04. Food for Pets/04. Food for Pets.py
days = int(input())
food = int(input())
dog_food = 0
cat_food = 0
cookies = 0

for day in range( 1, days + 1):
    if day % 3 == 0:
        dog = int(input())
        cat = int(input())
        dog_food += dog
        cat_food += cat
        cookies += (dog + cat)  / 10
        continue
    dog_food += int(input())
    cat_food += int(input())
    

all_eaten_food =  (dog_food + cat_food) / food * 100
eaten_food = (food - (food - (dog_food + cat_food)))  / 10
eaten_food_dog = dog_food / eaten_food * 10
eaten_food_cat = cat_food / eaten_food * 10

print(f'Total eaten biscuits: {round(cookies)}gr.')
print(f'{all_eaten_food:.2f}% of the food has been eaten.')
print(f'{eaten_food_dog:.2f}% eaten from the dog.')
print(f'{eaten_food_cat:.2f}% eaten from the cat.')

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam -  28 and 29 March 2020/04. Food for Pets/04. Food for Pets.js
function foodForPets(arg) {
    let days = Number(arg.shift())
    let food = parseFloat(arg.shift())
    let dogFood = 0
    let catFood = 0
    let cookies = 0

    for (let i = 1; i < days + 1; i++) {
        if (i % 3 == 0) {
            dog = Number(arg.shift())
            cat = Number(arg.shift())
            dogFood += dog
            catFood += cat
            cookies += (dog + cat) / 10
            continue
        }

        dogFood += Number(arg.shift())
        catFood += Number(arg.shift())
    }

    let allEatenFood = (dogFood + catFood) / food * 100
    let eatenFood = (food - (food - (dogFood + catFood))) / 10
    let eatenFoodDog = dogFood / eatenFood * 10
    let eatenFoodCat = catFood / eatenFood * 10

    console.log(`Total eaten biscuits: ${Math.round(cookies)}gr.`)
    console.log(`${allEatenFood.toFixed(2)}% of the food has been eaten.`)
    console.log(`${eatenFoodDog.toFixed(2)}% eaten from the dog.`)
    console.log(`${eatenFoodCat.toFixed(2)}% eaten from the cat.`)
}


// foodForPets([
//     3,
//     1000,
//     300,
//     20,
//     100,
//     30,
//     110,
//     40,
// ])